doc/api/n-api.md

```diff
@@ -525,6 +562,7 @@ This API can be called even if there is a pending JavaScript exception.
 #### napi_fatal_exception
 <!-- YAML
 added: v9.10.0
+napiExperimental: true
```

Comment:
I'm not sure how to best indicate these experimental APIs in the docs.  I still need to update the doc generation script to handle whatever option we decide on.

Reply:
What about using the `> Stability: 1 - Experimental` markers that the rest of the documentation uses? I wouldn’t mind having consistency with that…

Reply:
Yeah, that's roughly what I was thinking, I'll give it a try later today.

Reply:
@addaleax Is this what you had in mind?

Reply:
Yes :)